Traceback (most recent call last):
  File "C:\Users\tbeucler\.conda\envs\JB\lib\site-packages\jupyter_cache\executors\utils.py", line 58, in single_nb_execution
    executenb(
  File "C:\Users\tbeucler\.conda\envs\JB\lib\site-packages\nbclient\client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\Users\tbeucler\.conda\envs\JB\lib\site-packages\jupyter_core\utils\__init__.py", line 166, in wrapped
    return loop.run_until_complete(inner)
  File "C:\Users\tbeucler\.conda\envs\JB\lib\asyncio\base_events.py", line 647, in run_until_complete
    return future.result()
  File "C:\Users\tbeucler\.conda\envs\JB\lib\site-packages\nbclient\client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "C:\Users\tbeucler\.conda\envs\JB\lib\site-packages\nbclient\client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Users\tbeucler\.conda\envs\JB\lib\site-packages\nbclient\client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# Pooch Code
import pooch
datafile = pooch.retrieve('https://unils-my.sharepoint.com/:x:/g/personal/tom_beucler_unil_ch/Efg089STo25Gq6N_BBn_qGoBIsAOd2yUNBgeTfPR2wxw4g?download=1',
                          known_hash='84d455fb96dc8f782fba4b5fbe56cb8970cab678f07c766fcba1b1c4674de1b1')
------------------


[1;31m---------------------------------------------------------------------------[0m
[1;31mModuleNotFoundError[0m                       Traceback (most recent call last)
Cell [1;32mIn[1], line 2[0m
[0;32m      1[0m [38;5;66;03m# Pooch Code[39;00m
[1;32m----> 2[0m [38;5;28;01mimport[39;00m [38;5;21;01mpooch[39;00m
[0;32m      3[0m datafile [38;5;241m=[39m pooch[38;5;241m.[39mretrieve([38;5;124m'[39m[38;5;124mhttps://unils-my.sharepoint.com/:x:/g/personal/tom_beucler_unil_ch/Efg089STo25Gq6N_BBn_qGoBIsAOd2yUNBgeTfPR2wxw4g?download=1[39m[38;5;124m'[39m,
[0;32m      4[0m                           known_hash[38;5;241m=[39m[38;5;124m'[39m[38;5;124m84d455fb96dc8f782fba4b5fbe56cb8970cab678f07c766fcba1b1c4674de1b1[39m[38;5;124m'[39m)

[1;31mModuleNotFoundError[0m: No module named 'pooch'

